boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_defconfig

```diff
@@ -4,74 +4,19 @@ CONFIG_SOC_SERIES_NRF53X=y
 CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
 CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP=y
 
-CONFIG_BOARD_ENABLE_DCDC_APP=y
```

Comment:
Does the boardfiles need to change in this PR? 
If so, please see if you can use the changes from https://github.com/zephyrproject-rtos/zephyr/pull/49720.
This needs to be done with caution as we have large boardfile changes happening in two places.

Reply:
Another option is to bring the upstream boardfiles into NCS, temporarily.

Reply:
Yes , to remove those "=n" kconfig in child image(mcuboot) overlay, the boardfiles need to be changed.
As the comment in https://github.com/zephyrproject-rtos/zephyr/pull/49720.
> The basic default for a board needs to have all peripherals disabled and only allows for UART and timer to be enabled

What I do is to copy your changes of ``nrf5340_audio_dk_nrf5340_cpuapp_defconfig`` and ``nrf5340_audio_dk_nrf5340_cpunet_defconfig`` from upstream PR except license declaration.

Reply:
Great!